docs/src/pages/guides/notifications.md

```diff
@@ -1,3 +1,214 @@
+
+**Domain**
+A domain that you have set in mailgun. ex. `sandboxmail.mailgun.com`
+
+`Monika` filters the notification types for mailgun and breaks down the data configuration, setting up a mailgun service and then sending through the data using mailgun-js lib.
+
+## Sendgrid
+
+Similar to mailgun, sendgrid is also an email delivery service. You need to have a sendgrid account before you can send an email through this method. For sending a notification through sendgrid all you need to set is the API_KEY and recipient email addresses.
+
+```json
+{
+  "id": "unique-id",
+  "type": "sendgrid",
+  "data": {
+    "recipients": ["mantap@jiwa.com"],
+    "apiKey": "YOUR_API_KEY"
+  }
+}
+```
+
+**ID**
+Notification ID, to identify the notification number.
+
+**Type**
+Type of Notification in this case it would be set as `sendgrid`.
+
+**Data**
+The detail settings needed for sending email through sendgrid.
+
+**Recipients**
+Recipient email addresses, set as an array for sending to multiple addresses. ex. `["monika@testmail.com", "symon@testmail.com"]`
+
+**Api Key**
+The api key that sendgrid gives for your account.
+
+`Monika` is using @sendgrid/mail library for sending the sendgrid type notification. `Monika` will breakdown the configuration set the sengrid type API_KEY settings and send the message using the library.
+
+## SMTP
+
+SMTP (Simple Mail Transfer Protocol) is an email service using the TCP/IP protocol. For this you need to have an active email account to be used as transporter, a middle service that relays the email sending process.
+
+```json
+{
+  "id": "unique-id",
+  "type": "smtp",
+  "recipients": ["a@a.com"],
+  "data": {
+    "recipients": ["mantap@jiwa.com"],
+    "hostname": "smtp.mail.com",
+    "port": 222,
+    "username": "dennypradipta",
+    "password": "bismillah"
+  }
+}
+```
+
+**ID**
+Notification ID, to identify the notification number.
+
+**Type**
+Type of Notification in this case it would be set as `smtp`.
+
+**Data**
+The detail settings needed for sending email through smtp.
+
+**Recipients**
+Recipient email addresses, set as an array for sending to multiple addresses. ex. `["monika@testmail.com", "symon@testmail.com"]`
+
+**Hostname**
+The smtp host that you will be using for sending the email. ex. `smtp.gmail.com`
+
+**Port**
+The port that has been allowed to be used for sending mail in your host. ex. `587`
+
+**Username**
+The username that has been registered on your smtp server. ex. `yourusername@gmail.com`
+
+**Password**
+The password set for your username. ex. `thepasswordforyourusername`
+
+`Monika` will breakdown the configuration with type `smtp`, registering the username and password to the mail host with certain port and create a transporter to send to your recipient addresses.
+
+## Webhook
+
+Besides the email notifications, `Monika` also provide webhook service using REST API calls and POST methods.
```

Comment:
Besides the email notifications, Monika can also send the alerts via web hook. You need to define the URL and the HTTP method in Monika's configuration as follows.